.github/workflows/build-publish.yml

```
@@ -27,12 +27,17 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ github.token }}
 
+    - name: set date-based tag
+      run: |
+        echo "DATE_TAG=$(date +%Y-%m-%dT%H-%M)" >> $GITHUB_ENV
```

Given that the tag contains hours and minutes, let's call the variable `DATE_TIME_TAG`.

Although I couldn't provoke a collision with Bash-builtin syntax, I'd say it's better to be on the safe side and put the format string into single quotes.

```suggestion
        echo "DATE_TIME_TAG=$(date '+%Y-%m-%dT%H-%M')" >> $GITHUB_ENV
```